spec/system/subguide_cards_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'ruby-progressbar/outputs/null'
+
+RSpec.describe 'GuideCards', type: :system, js: true do
+  let(:subguide_card_fixture) { Rails.root.join('spec', 'fixtures', 'subguide_card_fixture.csv') }
+  before do
+    SubGuideLoadingService.new(csv_location: subguide_card_fixture,
+                               progressbar: ProgressBar.create(output: ProgressBar::Outputs::Null)).import
+  end
+
+  describe 'SubGuide page with image display' do
+    it 'shows all images for the SubGuide path' do
```

Comment:
I'm not sure this is true. The test doesn't check for image tags, for example. It appears to check for some kind of title or heading.

I understand that we don't want to run the card image loading service in tests, because it hits aws. Instead, to test the show page functionality, we could create a card image object with a false value and test that the image tag is produced on the page.